cucumber/authenticators_azure/features/authn_azure.feature

```diff
@@ -0,0 +1,49 @@
+Feature: Azure Authenticator - Users can authenticate with Azure authenticator
+
+  Background:
+    Given a policy:
+    """
+    - !policy
+      id: conjur/authn-azure/prod
+      body:
+      - !webservice
+
+      - !variable
+        id: provider-uri
+
+      - !group
+        id: apps
+
+      - !permit
+        role: !group apps
+        privilege: [ read, authenticate ]
+        resource: !webservice
+    """
+    And I am the super-user
+    And I successfully set Azure variables
+    And I have host "no-assigned-identity-app"
+    And I set Azure annotations to host "no-assigned-identity-app"
+    And I grant group "conjur/authn-azure/prod/apps" to host "no-assigned-identity-app"
+
+  Scenario: A valid Azure token to get Conjur access token
+    # We want to verify the returned access token is valid for retrieving a secret
```

Comment:
Instead of adding a comment here, I suggest you move it to the feature
`Feature: Azure Authenticator - Users can authenticate with Azure authenticator` ->
`Feature: Azure Authenticator - Users can authenticate with Azure authenticator and fetch secret`

Reply:
Renamed the scenario to `Users can authenticate with Azure authenticator and fetch secret`